### PR TITLE
[Token scanning-Phase 1-3] API keys profile page

### PR DESCRIPTION
**Pull request by zhhyu**
- API keys profile page shows the revoked API keys info.
image
Or
image

- The page to verify API keys shows "Link" rather than the original URL. 
image

### CODE REVIEW

File: src/NuGetGallery/Views/Users/ApiKeys.cshtml [hunk 1]
```diff
@@ -68,6 +68,35 @@
...
+        <!-- /ko -->
+        <!-- ko text: window.nuget.commaJoin(RevocationDescriptions().map(function(d) { return "'" + d + "'"; } )) --><!-- /ko -->
+        <!-- ko if: RevocationDescriptions().length === 1 -->
+        has
+        <!-- /ko -->
+        <!-- ko ifnot: RevocationDescriptions().length === 1 -->
+        have
+        <!-- /ko -->
+        been revoked.<br />
+        Please check the email from <<span>support@nuget.org</span>> for details, and regenerate or create
```
Comment (joelverhagen) [inline src/NuGetGallery/Views/Users/ApiKeys.cshtml]: The markup for this email is pretty weird. `&lt;support@nuget.org&gt;` is probably what you want. `&lt;` and `&gt;` are called HTML entities. By the way do we normally enclose our support email in angle brackets like this when display it? Seems a bit odd.

Is there a variable/config for `support@nuget.org`?

File: src/Bootstrap/less/theme/page-api-keys.less [hunk 2]
```diff
@@ -93,4 +93,8 @@
   {
     border-top: @section-border;
   }
+
+  .revoke-api-key {
+    color: red
```
Comment (joelverhagen) [inline src/Bootstrap/less/theme/page-api-keys.less]: Can you check if this is the error color we use in other contexts? Consider the color we use for `AlertError`.

Comment (zhhyu) [reply src/Bootstrap/less/theme/page-api-keys.less]: Use "brand-danger" now.

File: src/NuGetGallery/Views/Users/ApiKeys.cshtml (hunk 1 above)
Comment (zhhyu) [reply src/NuGetGallery/Views/Users/ApiKeys.cshtml]: Use Config variable and remove the angle brackets.

**Pull request merged**